include/amici/steadystateproblem.h

```diff
@@ -326,6 +367,8 @@ class SteadystateProblem {
     AmiVector delta;
     /** error weights */
     AmiVector ewt;
+    /** error weights for backward quadratures */
```

Comment:
state this has dimensions of parameters/quadratures, and that both of them are only serve as temporary storage during wrms computation?